cookbooks/steinberger/doc/steinberger.tex

```diff
@@ -0,0 +1,157 @@
+If the look-up table contains thermal expansivity and specific heat without the effect of phase transitions, then \aspect{} can compute latent heat effects based on the pressure and 
+temperature derivatives of the specific enthalpy (using the approach of \cite{nakagawa2009incorporating}).
+In our case, we do not include latent heat in our model, mainly because it can lead to numerical problems.
+So the look-up table is computed without latent heat effects, and we set the ``Latent heat'' parameter to false. 
+\lstinputlisting[language=prmfile]{cookbooks/steinberger/doc/lookup.part.prm.out}
+In a research application, it would be consistent with the projected density approximation (or any other compressible approximation) to compute latent heat. To avoid the numerical instabilities, either the resolution needs to be fine enough so that each phase transitions is resolved by several mesh cells, 
+or the energy equations needs to be solved for entropy instead of pressure (which is an option available in \aspect{}; in this case, the look-up table needs to be given in terms of entropy and pressure). 
+
+\paragraph{The look-up table format.}
+The format of these look-up table is implemented in the \href{https://aspect.geodynamics.org/doc/doxygen/classaspect_1_1MaterialModel_1_1MaterialUtilities_1_1Lookup_1_1MaterialLookup.html}{aspect::MaterialModel::MaterialUtilities::Lookup::MaterialLookup} class. 
+Two different formats are currently supported: Perple\_X and HeFESTo. 
+The format needs to be selected in the input file, and each format has a specific header and needs to be structured in a specific way. The paragraph below explains how to structure a Perple\_X file. 
+This file format is the default, it is also the more flexible format and it is what is used in this cookbook. Since the only requirements for the format are the header and the order of some of the columns, 
+files created with other mineral physics software can also be converted to this format.  
+
+The Perple\_X header contains the following in the first 13 lines: 
+The Perple\_X version, the name of the data table, the dimensions of the data table (for example, for a table with one dimension being pressure, the other temperature, this would be 2), the variable in the first dimension (this either needs to be `T(K)' for temperature, or `P(bar)' for pressure), the minimum value of this variable, the increments this variable will be increased with in the table, and the number of different values of this variable the table contains, the second variable (again with minimum, increment and number of values), the number of material properties in the table, and finally, the names of the columns. The first two columns need to be the pressure and temperature (in any order). The other required column names are:
```

Comment:
Can you just convert this into a numbered list? I think this would be easier to read